### PR TITLE
Replace "this technical specification" with "this document"

### PR DESCRIPTION
**Pull request by tkoeppe**

### CODE REVIEW

**Comment by tkoeppe** [comment]
@zygoloid, @jwakely, @jensmaurer: ISO has asked to use "this document" exclusively to refer to the current document. I would welcome your input if all the replacements still make sense, or if some wider rewording is warranted.

File: mods.html [hunk 1]
```diff
@@ -4,7 +4,7 @@ <h1>Modifications to the C++ Standard Library</h1>
   <cxx-section id="mods.general">
     <h1>General</h1>
     <p>
-      Implementations that conform to this technical specification shall
+      Implementations that conform to this document shall
```
Comment (jwakely) [inline mods.html:7]: I'm not sure what it means to conform to this document when we've already stated that it's non-normative, but I don't have a better suggestion right now. And I don't think it's a big deal to leave it like this.

**Review by jwakely — APPROVED** [review]

**Review by jensmaurer — APPROVED** [review]
Looks good.

File: mods.html (hunk 1 above)
Comment (tkoeppe) [reply mods.html:7]: Happily, ISO has separately asked to remove this sentence entirely (with an amusing explanation), so this problem will go away in the next commit.

Comment (tkoeppe) [reply mods.html:7]: ISO has separately requested that we remove the statement that we're non-normative (with an amusing explanation), so hopefully that resolves this tension.

**Comment by tkoeppe** [comment]
Many thanks!

**Review by zygoloid — COMMENTED** [review]
I'm a bit late to the party, but looks good to me too :)

**Comment by tkoeppe** [comment]
Great, thanks! Not too late, I've not sent the next revision yet :-)

**Pull request merged**